api/proto/rekor/v2/rekor_service.proto

```diff
@@ -29,6 +30,32 @@ option java_multiple_files = true;
 option java_outer_classname = "RekorV2Service";
 option ruby_package = "Sigstore::Rekor::V2";
 
+option (grpc.gateway.protoc_gen_openapiv2.options.openapiv2_swagger) = {
+  info: {
+    title: "Rekor v2";
+    version: "2.0";
+    contact: {
+      name: "Rekor v2 project";
+      url: "https://github.com/sigstore/rekor-tiles";
+      email: "sigstore-dev@googlegroups.com";
+    };
+    license: {
+      name: "Apache License 2.0";
+      url: "https://github.com/sigstore/rekor-tiles/blob/main/LICENSE";
+    };
+  };
+
+  host: "*.rekor.sigstore.dev";
+  external_docs: {
+    url: "https://github.com/sigstore/rekor-tiles";
```

Comment:
Would this point to README.md? I dunno if that even matters?

Reply:
The readme is the first thing you see when you visit that link